### DISCUSSION TITLE
I need help wit the bookmark

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#71] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
The bookmark doesnt work for me i think my school blocked it somehow but i need help

**Comment by ghost** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/77454237/196487059-b52bfb60-1364-4738-8e93-216702cae687.png)
how do i reenable javascript

**Reply by Gumkid3** [reply]
its not ur school, its bc of the new chrome os update that patched it

**Reply by ghost** [reply]
is there a away to downgrade my chrome

**Reply by 3kh0** [reply]
@ttvbeghhf use https://chrome100.dev

**Reply by ghost** [reply]
how do i use it

**Reply by 3kh0** [reply]
@ttvbeghhf All of the instructions are there, READ THEM!

**Reply by ghost** [reply]
my laptop i need to requst to add an extenison

**Reply by 3kh0** [reply]
@ttvbeghhf I dont know how to help you there
